eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -28,6 +29,13 @@ opts="
     --env=ZENKO_SESSION_TOKEN=${ZENKO_SESSION_TOKEN}
     --env=TOKEN=${TOKEN}
     --env=STAGE=${STAGE}
+    --env=CYPRESS_KEYCLOAK_USER_FULLNAME=${OIDC_FULLNAME}
```

Comment:
this will need to be quoted since there is a space  in the value
```suggestion
    --env=CYPRESS_KEYCLOAK_USER_FULLNAME="${OIDC_FULLNAME}"
```

Reply:
thank you!